python/neuroglancer/pipeline/task_queue.py

```diff
@@ -98,10 +161,20 @@ def insert(self, task):
             "tag": task.__class__.__name__
         }
 
-        self.api.insert(project=self._project,
-                        taskqueue=self._queue_name,
-                        body=body).execute(num_retries=6)
+        def cloud_insertion(api):
```

Comment:
The point of a closure is that it is 1) shielded from being misinterpreted as a method that is used elsewhere 2) to aid in the ease of function construction. It looks weird at first, but you get used to it.